packages/forms-web-app/Dockerfile

```diff
@@ -15,19 +15,24 @@ RUN npm run build
 # Stage 2/3: Run tests
 FROM node:14-alpine AS test
 
+WORKDIR /opt/app
+
 COPY --from=build /opt/app ./
 COPY --from=build /opt/common ../common
 
 COPY packages/forms-web-app/__tests__ ./__tests__
 COPY packages/forms-web-app/jest.config.js jest.config.js
-COPY packages/forms-web-app/.eslintrc .eslintrc
+COPY packages/forms-web-app/.eslint* ./
+COPY packages/forms-web-app/.prettier* ./
 
 RUN npm run lint
 RUN npm run documentation; echo $? > /npm.exitcode
```

Comment:
Same here - doesn't this swallow the error?  We want test failures to fail the build